Source/santad/SNTDaemonControlController.m

```diff
@@ -223,4 +232,56 @@ - (void)postRuleSyncNotificationWithCustomMessage:(NSString *)message reply:(voi
   reply();
 }
 
+#pragma mark Bundle ops
+
+///
+///  This method is only used for santactl's bundleinfo command. For blocked executions, Santa GUI
+///  calls on santabs directly.
+///
+///  Hash a bundle for an event. The SNTBundleHashBlock will be called with nil parameters if a
+///  failure or cancelation occurs.
+///
+///  @param event The event that includes the fileBundlePath to be hashed.
+///  @param reply A SNTBundleHashBlock to be executed upon completion or cancelation.
+///
+///  @note If there is a current NSProgress when called this method will report back it's progress.
+///
+- (void)hashBundleBinariesForEvent:(SNTStoredEvent *)event
+                             reply:(SNTBundleHashBlock)reply {
+  SNTXPCConnection *bs = [[SNTXPCConnection alloc] initClientWithServiceName:@"com.google.santabs"];
```

Comment:
Move the service ID to a serviceId class method in SNTXPCBundleServiceInterface, following the example of SNTXPCControlInterface